plugin/templates/_package.json

```diff
@@ -10,7 +10,7 @@
   "author": "<%= userName %>",
   "main": "lib/index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha tests --recursive"
```

Comment:
`mocha tests/` will look for `.js` files in the `tests` directory but will not look in directories inside of the `tests` directory. To put it another way:

`mocha tests/` translates to `mocha tests/*.js`
`mocha tests --recursive` translates to `mocha tests/**/*.js`